wercker.yml

```diff
@@ -59,3 +63,46 @@ push:
         tag: $VERSION
         entrypoint: /oci-cloud-controller-manager
         user: 65534 # nobody
+
+release:
+  box:
+    id: oraclelinux:7-slim
+  steps:
+    - script:
+        name: set ENV vars
+        code: |
+          export VERSION=$(cat VERSION.txt)
```

Comment:
Should this be 

export VERSION=$(GITHUB_RELEASE_VERSION)

Reply:
Nope it pick up the version